src/components/AppSidebar/AppSidebar.vue

```diff
@@ -131,7 +131,8 @@
 				class="app-sidebar-header">
 				<!-- close sidebar button -->
 				<a
-					v-tooltip.auto="t('close')"
+					v-tooltip.auto="
+						t('Close')"
```

Comment:
Seems weird, and most likely invalid with our styleguide :thinking:

Reply:
Yeah, however linter does not complain... 🤔 

Alternative approach would be this, is just a bit more of code...
```
data {
  return {
    closeTranslation: t('Close')
  }
}
```

Reply:
Why do you need the newline anyway?

Reply:
`npm run l10n:extract` respectively `gettext-extractor` ([build/extract-l10n.js](https://github.com/nextcloud/nextcloud-vue/blob/9526649a798f259d2c637e4623bb5f9b609b410b/build/extract-l10n.js)) did not recognise it within an html-tag. So no extraction, no translations on transifex... 😢

Reply:
Can't we fix the `gettext-extractor` directly? No one is going to remember putting new lines :/

Reply:
I didn't find a way... A PR on the repo seems not worth the work, as there are other PRs that just didn't get merged (see linked issue above and [this PR](https://github.com/lukasgeiter/gettext-extractor/pull/44)).
Only other way, that i could imagine would be to use a html-parser with specific html-tags t and n to put the translations in a second time, but thats even more hacky.
Or just some completely different parser-package... 🤔

Reply:
Any preferences? As i said - i don't see us currently fixing the extractor.

- Using the given way or the proposed data/prop way? The latter is probably a bit cleaner, but 5 LoC more. The prop could also be done as object `l10n.close = t('Close')`
- Just came to my mind that i could additionally add a workflow spitting a warning if some pattern `="t(` is found. This is how i found those three occurences.

Reply:
> data {
>   return {
>     closeTranslation: t('Close')
>   }
> }

Let's use this syntax instead then, will be cleaner and easier to manage

Reply:
Ok, so it's this version now.

The webpack-dev-build i looked into for moment, but it seems to be not fully straigth forward.